### ISSUE TITLE
Keyboard layout option disabled (Alt + Shift not working)

### ISSUE THREAD

**Issue by Maeril** [hmlendea/geforcenow-electron#76]
Hi,

Ever since I updated to the latest version (same problem on 1.1.5, then 1.1.6) the keyboard layout option is grayed out in the session's settings. 

Plus, pressing Alt + Shift in-session doesn't do anything.

This is particularly hindering because I have an AZERTY keyboard. :( 

I'm on Ubuntu 20.04.

**Comment by hmlendea** [comment]
Yeah, unfortunately the trick about changing the UserAgent to a ChromeOS one doesn't see to work anymore to work around this issue :(

**Comment by dirksalewski** [comment]
I'm not sure if this can help, but with 

_Mozilla/5.0 (Windows NT 10.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.7113.93 Safari/537.36_

I can at least acess the web version. Maybe the UserAgent has to be changed to that?
